test/instrumentations/next/app/src/app/globals.css

```diff
@@ -0,0 +1,27 @@
+@tailwind base;
```

Comment:
In general, I'd like us to have as few files as possible related to test apps.
Let's remove this and `favicon.ico` too

Reply:
Done